DRAFT_climate_indicators.Rmd

```diff
@@ -0,0 +1,1255 @@
+
+### Original units
+
+Varied. Specified below for each parameter.
+
+### Temporal coverage
+
+1957 - present
+
+### Additional comments about the data set
+
+The data format has recently changed from .BIL to .nc (netcdf) and now a single file contains all the rasters for one year (365 days), and sometimes for multiple variables also.
+
+## Ecosystem characteristic
+
+### Norwegian standard
+
+These variables typically will fall under the *abiotiske egenskaper* class.
+
+### SEEA EA
+
+In SEEA EA, these variables will typically fall under A1 - Physical state characteristics.
+
+## Collinearities with other indicators
+
+Climate variables are most likely to be correlated with each other (e.g. temperature and snow). Also, some climate variables are better classed as pressure indicators, and these might have a causal association with several condition indicators.
+
+## Reference condition and values
+
+### Reference condition
+
+The reference condition for climate variables is defined as the last normal period 1961-1990.
+
+### Reference values, thresholds for defining *good ecological condition*, minimum and/or maximum values
+
+-   Un-scaled indicator value = median value over 5 year periods
+
+-   Upper reference level (best possible condition) = median value from the reference period
+
+-   Thresholds for good ecosystem condition = 2 standard deviation units away from the upper reference level for the climate variable during the reference period.
+
+-   Lower reference values (two-directional) = 5 standard deviation units for the climate variable during the reference period (implies linear scaling).
+
+## Uncertainties
+
+For the indicator map (1 x 1 km raster) there is no uncertainty associated with the indicator values. For aggregated indicator values (e.g. for regions), the uncertainty in the indicator value is calculated from the spatial variation in the indicator values via bootstrapping. This might, however, be changed later to the temporal variation between the five years of each period.
+
+## References
+
+<https://senorge.no/>
+
+*rr and tm are being download from:* <https://thredds.met.no/thredds/catalog/senorge/seNorge_2018/Archive/catalog.html>
+
+### Additional resources
+
+[Stars package](https://r-spatial.github.io/stars/)
+
+[R as a GIS for economists](https://tmieno2.github.io/R-as-GIS-for-Economists/stars-basics.html) chapter 7
+
+## --------------------------------------------------
+
+## Analyses
+
+### Data set
+
+The data is downloaded to a local NINA server, and updated regularly.
+
+```{r}
+path <- ifelse(dir == "C:", 
+      "R:/GeoSpatialData/Meteorology/Norway_SeNorge2018_v22.09/Original",
+      "/data/R/GeoSpatialData/Meteorology/Norway_SeNorge2018_v22.09/Original")
+```
+
+This folder contains folder for the different parameters
+
+```{r}
+(files <- list.files(path))
+```
+
+This table explains them in more detail
```

Comment:
This table only contains variables related to snow. I guess the data contain other classes of variables, too (temperature, wind, pressure)? If the table only shows a subset of the data, that should be stated. Or a table showing all variables should be given.

Reply:
The table initially shows rows 1-10, but there are more variables on page 2 (you can switch between pages at the bottom of the table)